manually_build_docs.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
You could put this in the docsrc folder in order for it not to be in the root.

Reply:
Good idea. I didn't really want to commit it as fake build should be enough, but doesn't work for me right now. It still has value to show simpler steps, bit should eventually be removed.

Reply:
Done, thanks!